MANIFEST.in

```diff
@@ -5,6 +5,9 @@ include *.txt
 include buildout.cfg
 include tox.ini
 
+recursive-include src *.pt
```

Comment:
Here's something I REALLY don't like: I curate `MANIFEST.in` after running `config-package.py` to remove all entries that are not relevant. That way we don't get that stupid warning every time the buildout is run or when the package is installed about no matching files for that specification. Now those entries are back, and so is the warning message.

Reply:
I was wondering where these lines came from.

But you are right: The default `MANIFEST.in` configuration in `meta/config` has probably too many hard coded entries. We could remove at least some of them. As we have linting for `MANIFEST.in` in place (which at least checks if too little is included) it does not hurt too much to reduce the default rules. Only the next time a package needing removed rules it has to be updated to include them via `.meta.toml`. – A PR is welcome, I do not like theses warnings either as they look very professional.

Reply:
PR open at https://github.com/zopefoundation/meta/pull/86